### ISSUE TITLE
Shopping cart glitch.

### ISSUE THREAD

**Issue by YourWaltNightmare** [Malcolm-Q/LC-LateGameUpgrades#340]
Most of the time, when I grab shopping cart, its has glitched state.
Here is a video of an issue.

https://github.com/Malcolm-Q/LC-LateGameUpgrades/assets/109439715/b96a3d35-0b1e-49f2-8d00-3bcc1d58316c

**Comment by YourWaltNightmare** [comment]
Ok. LGU has been updated recently. I'll see if the problem still exist.

**Comment by YourWaltNightmare** [comment]
Nope. Still an issue. It happens randomly.

**Comment by WhiteSpike** [comment]
Provide me your logs and I will see what I can do.

**Comment by YourWaltNightmare** [comment]
[LogOutput.log](https://github.com/Malcolm-Q/LC-LateGameUpgrades/files/14452009/LogOutput.log)
My last solo session. I quit the game exactly after cart was broken again.

**Comment by WhiteSpike** [comment]
Seems like a silent error and the player's animator just breaks mid animation. 
TL;DR Seems like a vanilla issue as LGU doesn't make any changes in the game's animations, it just uses them whenever available.
So if anything, it's either one of the mods that changes how the player's animations work or it's a vanilla issue entirely that is sometimes triggered and is most noticeable with the shopping cart.

**Comment by YourWaltNightmare** [comment]
I know this issue is outdated but if it'll help you - shiplobby cause these animations issues and some different problems (its also outdated like 4 month ago)
